src/authorizations/components/redesigned/CustomApiTokenOverlay.tsx

```diff
@@ -41,6 +41,7 @@ import {getResourcesStatus} from 'src/resources/selectors/getResourcesStatus'
 import {formatApiPermissions} from 'src/authorizations/utils/permissions'
 
 import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+import {capitalize} from 'lodash'
```

Comment:
nit: lodash is a library, can we move this import up to all the library imports

Reply:
Absolutely!